src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -398,6 +398,25 @@ await TestAsync(markup,
             Documentation("summary for interface IGoo"));
     }
 
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/60725")]
+    public async Task TestMergedRemarks()
+    {
+        var markup =
+            """
+            /// <remarks>hello1</remarks>
+            public partial class C {
+            }
+
+            /// <remarks>hello2</remarks>
+            public partial class $$C {
+            }
+            """;
+
+        await TestAsync(markup,
+            MainDescription("class C"),
+            Remarks("hello1 hello2"));
```

Comment:
>Remarks("hello1 hello2")); [](http://example.com/codeflow?start=12&length=26)

Is this ordering guaranteed?

Reply:
within the bounds of this file yes.  compiler will keep declarations sorted in the same order in the same file.